internal/admin/dashboard/templates/index.html

```
@@ -484,27 +484,41 @@ <h5>Body</h5>
            aria-modal="true"
            :aria-hidden="(!conversationOpen).toString()"
-           aria-labelledby="conversation-drawer-title"
-           aria-describedby="conversation-drawer-content"
+           aria-labelledby="interactions-drawer-title"
+           aria-describedby="interactions-drawer-content"
            @keydown.escape.window="conversationOpen && closeConversation()">
         <div class="conversation-drawer-header">
-            <h3 id="conversation-drawer-title">Conversation</h3>
-            <button class="pagination-btn" x-ref="conversationCloseBtn" aria-label="Close conversation" @click="closeConversation()">Close</button>
+            <h3 id="interactions-drawer-title">Interactions</h3>
+            <button class="pagination-btn" x-ref="conversationCloseBtn" aria-label="Close interactions" @click="closeConversation()">Close</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)